### DISCUSSION TITLE
Printing the results of sopm

### DISCUSSION THREAD

**Discussion by SudoWodo** [SudoWodo/StochOptim#17] [Q&A] [answered]
Tibbles have nice printing method that conforms to the screen size automatically. Should we use that to print output of sopm?

**Comment by nashjc** [comment, marked as answer]
Each use of things outside the base language mean the package has to be adjusted if the dependencies change. So I'd be very careful about using tidyverse material. In some cases it is the right choice, but here definitely not. It may be that the R6 usage is similarly a long-term inconvenience, though a work-around for that may be much less easy to find than one for printing.

With the printing, I suggest setting a target width (e.g. 80 characters) and working with that.

J

**Comment by hwborchers** [comment]
When a user of this package wants, he can say `as.tibble(res)` and print it. No need whatsoever to include this in the package. Please work on the basic functionality of 'StochOptim', I will be glad when I see the first data frames being returned.
BTW, do not fiddle around with generating vignettes. Generating help pages is important, vignettes are not. Later on, any valid R Markdown document can be converted to a vignette without much extra work.
